content/articles/hn-first-place-stats.md

```diff
@@ -0,0 +1,32 @@
+Title: The Hacker News effect and what we can learn from it
+Date: 2015-06-08
+Short_summary: Looking at the stats from being at the first spot on HN
+Category: Misc
+Authors: Vincent
+
+
+*Reading time: ~10 minutes.*
+
+Last week my article [Using Protobuf instead of JSON to communicate with a frontend](https://blog.wearewizards.io/using-protobuf-instead-of-json-to-communicate-with-a-frontend) ended up being number one on [Hacker News](https://news.ycombinator.com) and we saw a crazy amount of traffic coming on this blog. 
+
+I thought that it would be interesting to analyze some of that data so here it is.
+<!-- PELICAN_END_SUMMARY -->
+
+## The data
+It is coming from Google Analytics and while I could just paste screenshots, that wouldn't be any fun so I exported some of it as CSVs and used them in the notebook you can see below. The data is over the past month to have more insights on various social "strategies" (or rather, lack of) as you will see in a bit.  
+
+One important thing that the notebook doesn't talk about is the average session duration since I forgot to include it (bear with me I'm writing this from a bed on a sunday afternoon/evening).
+
+In short, the average session duration is ~20s which is pretty bad but makes sense since most people might not be interested in a given article after reading a couple of sentences and just opened it because it was at the top. Interestingly enough, people stay longer (around 1:10) on [our website](https://wearewizards.io/). 
```

Comment:
that is a very good point actually, apparently GA cannot calculate time on page for single-page session so unless the reader hit a button or something, it will be almost 0 since we don't really have engagement hits any.

I'll remove that paragraph, makes actually more sense after reading about it